src/Library/demos/Launcher/main.js

```diff
@@ -3,14 +3,6 @@ import Gdk from "gi://Gdk";
 import Gio from "gi://Gio";
 import GLib from "gi://GLib";
 
-const launch_file = workbench.builder.get_object("launch_file");
-const file_name = workbench.builder.get_object("file_name");
-const file_location = workbench.builder.get_object("file_location");
-const change_file = workbench.builder.get_object("change_file");
-const uri_launch = workbench.builder.get_object("uri_launch");
-const uri_details = workbench.builder.get_object("uri_details");
-const change_uri = workbench.builder.get_object("change_uri");
```

Comment:
the `Gio._promisify` should be directly below the import statements as they related to them

Reply:
Added to the style guide.